MAINTAINERS.md

```diff
@@ -0,0 +1,59 @@
+# Maintainers
+
+This document contains list of present maintainers, their role and process to add new maintainers.
+
+## Current Maintainers
+
+|   Username    |    Focus Area   |
+| ------------- | -------------   |
+|  MarcoFalke   | General, QA     |
+|  fanquake     | General, Build  |
+|  hebasto      | General, UI/UX  |
+|  achow101     | General, Wallet |
+|  glozow       | General, Mempool|
+
+## Responsibilities and roles
+
+Project maintainers have commit access and are responsible for merging patches
+from contributors. They perform a janitorial role merging patches that the team
+agrees should be merged. Maintainers make sure that patches are safe and align
```

Comment:
perhaps: "They perform a janitorial role of implementing, by merging patches, the rough consensus of reviewers. Maintainers also make sure..."

Reply:
I used this sentence, let me know if it conveys the correct meaning: 

"They perform a janitorial role of implementing the rough consensus of reviewers by merging patches."